src/SIL.XForge/Realtime/RealtimeServiceCollectionExtensions.cs

```diff
@@ -28,6 +28,13 @@ public static class RealtimeServiceCollectionExtensions
                 options.NodeAndV8Options = nodeOptions;
             }
         });
+
+        // The debugger requires the timeout to be disabled
```

Comment:
It may obscure what is being done to say the debugger _requires_ the timeout to be disabled (eg maybe `--inspect` is being confused for `--inspect-brk`). If my other comment about this is accurate, what do you think about saying something like
> // Disable timeout so the debugger can be paused

Reply:
That is much better wording - thank you!